src/directive/sort-by.directive.ts

```diff
@@ -27,16 +28,12 @@ export class JhiSortByDirective implements AfterViewInit {
 
     @Input() jhiSortBy: string;
 
-    sortAscIcon = 'fa-sort-up';
-    sortDescIcon = 'fa-sort-down';
-
     jhiSort: JhiSortDirective;
+    config: JhiModuleConfig;
 
-    constructor(@Host() jhiSort: JhiSortDirective, private el: ElementRef, private renderer: Renderer, configService: JhiConfigService) {
+    constructor(@Host() jhiSort: JhiSortDirective, private el: ElementRef, renderer: Renderer, configService: JhiConfigService) {
```

Comment:
Since the `renderer` usage is removed. we can remove it here too

Reply:
it's done